api/health_check.proto

```diff
@@ -88,6 +88,14 @@ message HealthCheck {
   message RedisHealthCheck {
   }
 
+  // [#not-implemented-hide:] grpc.health.v1.Health-based healthcheck.
+  // See `GRPC doc <https://github.com/grpc/grpc/blob/master/doc/health-checking.md>`
```

Comment:
I think you need a trailing underscore for this to render as an external link. Please validate the docs output for this by temporarily deleting the #not-implemented-hide and viewing docs in a browser.

Reply:
Thinks, fixed. I've looked through uncommented html, now looks fine:)